tests/baselines/reference/user/npm.log

```diff
@@ -83,6 +83,7 @@ node_modules/npm/lib/cache.js(16,34): error TS2339: Property 'commands' does not
 node_modules/npm/lib/cache.js(49,30): error TS2339: Property 'prefix' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/cache.js(69,35): error TS2339: Property 'cache' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/lib/cache.js(70,12): error TS2339: Property 'config' does not exist on type 'typeof EventEmitter'.
+node_modules/npm/lib/cache.js(82,3): error TS2775: Assertions require every name in the call target to be declared with an explicit type annotation.
```

Comment:
haha wow this is going to be *super* annoying for anybody who turns on checkJS and then uses `assert` from `@types/node`. Hope this is only with strictNullChecks on.

Reply:
later: no this is a bug when importing assertions via commonjs.

Filed https://github.com/microsoft/TypeScript/issues/38379 to track it.